static/js/settings_display.js

```diff
@@ -184,6 +184,30 @@ exports.set_up = function () {
             },
         });
     });
+
+    $("#translate_emoticons").change(function () {
+        var data = {};
+        var setting_value = $("#translate_emoticons").is(":checked");
+        data.translate_emoticons = JSON.stringify(setting_value);
+        var context = {};
+        if (data.translate_emoticons === "true") {
+            context.new_mode = 'be';
+        } else {
+            context.new_mode = 'not be';
+        }
```

Comment:
I think we need these strings tagged for translation?  This whole system is annoying and we're hoping to change, but I think that's correct.

Reply:
@timabbott OK, I rebased the commit and they are tagged for translation now.